app/controllers/events_controller.rb

```diff
@@ -227,6 +228,7 @@ def event_params
                                     :enddate,
                                     :initial_value,
                                     :gameday_duration,
+                                    :has_place_3_match,
```

Comment:
I hate that this is getting so big but I have no idea how to fix it other than by creating multiple controllers for different event kinds. Which we agreed on not doing, because it would also mean that STI was probably not the right choice, which we didn't wanna change... arrrg.
Well, I suppose I can live with it.